### PR TITLE
[ONNX] Fix export issue of aten::layer_norm in opset 17

### PR DESCRIPTION
**Pull request by CYuxian**
For torch.nn.LayerNorm, weight and bias could be None(when parameter elementwise_affine is False or bias is False), but for onnx op LayerNormalization from opset 17, weight and bias cannot be None.

### CODE REVIEW

File: torch/onnx/symbolic_opset17.py [hunk 1]
```diff
@@ -47,6 +47,16 @@ def layer_norm(
     # layer_norm normalizes on the last D dimensions,
     # where D is the size of normalized_shape
     axis = -len(normalized_shape)
+    scalar_type = _type_utils.JitScalarType.from_value(input)
+    dtype = (
+        _type_utils.JitScalarType.FLOAT if scalar_type is None else scalar_type.dtype()
+    )
```
Comment (thiagocrepaldi) [inline torch/onnx/symbolic_opset17.py:53]:
```suggestion
    scalar_type = _type_utils.JitScalarType.from_value(input, _type_utils.JitScalarType.FLOAT)
    dtype = scalar_type.dtype()
```
Would assume `FLOAT` the correct value? what if this is executed under an Autocast region? couldn't the right type would be float16 or bfloat16?

**Review by thiagocrepaldi — APPROVED** [review]
LGTM.

File: torch/onnx/symbolic_opset17.py [hunk 2]
```diff
@@ -47,6 +47,14 @@ def layer_norm(
     # layer_norm normalizes on the last D dimensions,
     # where D is the size of normalized_shape
     axis = -len(normalized_shape)
+    scalar_type = _type_utils.JitScalarType.from_value(input, _type_utils.JitScalarType.FLOAT)
```
Comment (thiagocrepaldi) [inline torch/onnx/symbolic_opset17.py:50]: Would assume FLOAT the correct value? what if this is executed under an Autocast region? couldn't the right type would be float16 or bfloat16?

Comment (CYuxian) [reply torch/onnx/symbolic_opset17.py:50]: If the JitScalarType cannot be fetched from value, FLOAT will be used here. Maybe it's better not to use FLOAT as default? Just raise error if from_value returns None?

Comment (thiagocrepaldi) [inline torch/onnx/symbolic_opset17.py:50]:
```suggestion
    scalar_type = _type_utils.JitScalarType.from_value(
        input, _type_utils.JitScalarType.FLOAT
    )
```

Comment (thiagocrepaldi) [reply torch/onnx/symbolic_opset17.py]: We use `FLOAT` by default in other ops. I think we can give it a try the way it is

**Review by thiagocrepaldi — APPROVED** [review]

**Pull request closed without merging**